### ISSUE TITLE
ognn.loss.shapenet_loss grad loss part.

### ISSUE THREAD

**Issue by RunyiYang** [octree-nn/ognn-pytorch#6]
Thank you Dr. Wang for the great contribution of ognn and ocnn. While I use the ognn to train a vae using sdf, I observed that the loss is pretty hard to converge. It mainly because there is an extreme large term in the loss which is grad_loss of sdf. 

The current grad_loss is to compute the 
`(grad - grad_gt).pow(2).mean()`

which could lead to an extreme large number of the loss, e.g. 1e10 at some time. Should we consider to normalize the grad loss by using
`(grad - grad_gt).norm(2, dim=-1).mean()`

**Comment by wang-ps** [comment]
Thanks for your suggestions.  The loss you mentioned may work, too.
